src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$position-start-hand: rotate(-90deg);
+$border-radius-hand: 6px;
+$color-second-hand: #00539cff;
+$left-hand-position: 250px;
+$width-minute-hand: 170px;
+$height-minute-hand: 9px;
+$width-second-hand: 220px;
+$height-second-hand: 6px;
+$duration-minute-hand: 3600s;
+$duration-second-hand: 60s;
+$watch-height: 500px;
+$watch-width: 500px;
```

Comment:
p.10 But don't overuse them, don't create variable for the value that's used just once. (optional)